### DISCUSSION TITLE
Unsafe assignment of an `any` value - Resolver

### DISCUSSION THREAD

**Discussion by KevinAHarley** [react-hook-form/react-hook-form#9092] [Q&A]
Hey,

I am currently moving my project from formik to RHF. Everything has been fine until I added a resolver in the useForm hook.

yup: v0.32.11
react-hook-form: v7.36.0
hookform/resolvers: v2.9.8

`  const {
    control,
    handleSubmit,
    formState: { errors },
    clearErrors,
    setValue,
    getValues,
    watch,
  } = useForm<SignUpFormTypes>({
    defaultValues,
    mode: 'onBlur',
    resolver: yupResolver(validationSchema),
  })
`

On the resolver line, I get this message:
(property) resolver?: Resolver<SignUpFormTypes, any> | undefined
Unsafe assignment of an `any` value

Here is my type file:
`
export type SignUpFormTypes = {
  name: string
  surname: string
  email: string
  password: string
  avatar?: {
    uri: string
  }
}
`

**Comment by leapful** [comment]
@KevinAHarley : Please provide a codesandbox url for your problem.

Based on the source code you've posted, I will take a guess that you're missing type definition for yup schema "validationSchema" but it would be easy to find out the problem with codesandbox url sample.
